resources/views/clans.blade.php

```diff
@@ -55,12 +55,12 @@ class="fas fa-sort"></i>
                                     <tr class="data_row">
                                         <th scope="row">{{ $clan['rank'] }}</th>
                                         <td>{!! $clan['color_tag'] !!}</td>
-                                        <td>{{ $clan['name'] }}</td>
+                                        <td style="cursor: pointer" onclick="showClan('{{$clan["tag"]}}');">{{ $clan['name'] }}</td>
                                         <td>{{ number_format($clan['KDR'], 2) }}</td>
                                         <td>{{ sizeof($clan['leaders']) }}</td>
                                         <td>{{ sizeof($clan['members']) }}</td>
-                                        <td> {{ date("d/m/Y",$clan['founded'] / 1000) }}</td>
-                                        <td>{{ date("d/m/Y - H:i", $clan['last_used'] / 1000) }}</td>
+                                        <td> {{ \App\Utils::formatDate($clan['founded']) }}</td>
+                                        <td>{{ \App\Utils::formatDateTime($clan['last_used']) }}</td>
```

Comment:
Weird.
Can you replace it to just `Utils::formatDate...`?
Try to add `use \App\Utils;` at the top of the page.

Reply:
I thought last seen could show date and time. And founded just the date.
So 2 methods.